src/_data/toc/operations.yml

```diff
@@ -221,7 +221,7 @@ pages:
     - label: Security Scan
       url: "/magento/security-scan.html"
     - label: Security Best Practices
```

Comment:
Perhaps we should take this out of the TOC and make sure to add a description of the document and a link to the main security topic: https://docs.magento.com/user-guide/stores/security.html

Reply:
Fixed.